src/ensembl/doc/architecture/decisions/0004-unit-tests.md

```diff
@@ -12,18 +12,22 @@ When changes are made after the initial implementation of components, there is a
 
 ## Decision
 
-Write unit tests using Jest and Enzyme. Include snapshot testing too, for additional testing.
+Write unit tests using Jest and Enzyme.
 
-[Jest](https://jestjs.io/) is used as the main testing framework. Snapshot testing, which is a feature of Jest, is also used.
+[Jest](https://jestjs.io/) is used as the main testing framework and [`ts-jest`](https://kulshekhar.github.io/ts-jest/) is used as the TypeScrpit preprocessor to run them..
 
-[Enzyme](https://airbnb.io/enzyme/) is used as a utility of Jest to test React components.
+[Enzyme](https://airbnb.io/enzyme/) is used as a utility of Jest to test React components. A recent and promising alternative library for unit-testing React is react-testing-library. Its philosophy differs from Enzyme in that it offers the developer a very limited API and encourages testing of components only after they have been mounted in the DOM.
```

Comment:
One last remark is that the first sentence in this paragraph says that Enzyme is a utility of Jest, which isn't technically correct. Enzyme is just a library for testing React components, and Jest is just a test runner coupled with an assertion library (based on Jasmine, if I am not mistaken), and they don't really care about one another.

Reply:
Ah, I altered what was mentioned in the Enzyme website, but added Jest into the mix and made it wrong. I'll change this to be something meaningful.

Reply:
Made the changes in 469d5c8